extensions/todolist@bsaleil.org/Makefile.am

```diff
@@ -0,0 +1,14 @@
+todoschemadir = $(extensiondir)/schemas
```

Comment:
And, of course, remove the invocation of `glib-compile-schemas` from the build system.